models/fixtures/user.yml

```diff
@@ -186,14 +186,14 @@
   lower_name: user11
   name: user11
   full_name: User Eleven
-  email: user11@example.com
+  email: user011@example.com
```

Comment:
why change ?

Reply:
Because in the `email_1ddress.yml` fixtures, the `user11@example.com` is registered to `user1`, and changing that file rather than this, brings more issues and places where it should be updated (I initially went through that route with commits bdd4bc2d827fce00328493823fe3bd9e0359d73a and 77cd65708e5e56a11ae9017123a77ae35f4fe7fb), and decided later that the minimal impact would be to update the email address here.